tests/test_commands.py

```diff
@@ -2136,7 +2137,7 @@ def test_zrangestore(self, r):
         assert r.zrangestore("b", "a", "[a2", "(a3", bylex=True, offset=0, num=1)
         assert r.zrange("b", 0, -1) == [b"a2"]
 
-    @skip_if_server_version_lt("2.8.9")
```

Comment:
Some tests like this have varios decorators. In this case, it is ```skip_if_server_version_lt```. I thought if we are going to skip these tests, we should skip them entirely without checking redis version. That is why I removed these decorators and added just ```pytest.mark.skip```. Is that a good idea @chayim?